hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheRemoveBackupOperation.java

```diff
@@ -47,17 +47,13 @@ public CacheRemoveBackupOperation(String name, Data key, boolean wanOriginated)
     }
 
     @Override
-    public void run()
+    public void runInternal()
             throws Exception {
-        if (cache != null) {
-            response = cache.removeRecord(key);
-        } else {
-            response = Boolean.FALSE;
-        }
+        cache.removeRecord(key);
```

Comment:
there was a copy&paste error I fixed - response is irrelevant for backup operations.